server/stream.go

```diff
@@ -3029,6 +3042,7 @@ func (mset *stream) stateWithDetail(details bool) StreamState {
 	if !details {
 		state.Deleted = nil
 	}
+	state.Consumers += ndc
```

Comment:
so here Consumers counter will be lets say 10 but we wil not see those in consumer list still which would be confusing don't you think? Definitely welcome more visibility but maybe an additional key on state would be good (ugh), somthing for next release perhaps.

Reply:
Should I make it separate in State?